src/__tests__/proposals.int.test.ts

```diff
@@ -723,4 +740,167 @@ describe('/proposals', () => {
 			});
 		});
 	});
+
+	describe('POST /:proposalId/organizations', () => {
+		it('creates exactly one OrganizationProposal', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await insertTestOrganizations();
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			const before = await loadTableMetrics('organizations_proposals');
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					organizationId: 1,
+					proposalId: 1,
+				})
+				.expect(201);
+			const after = await loadTableMetrics('organizations_proposals');
+			expect(before.count).toEqual(0);
+			expect(result.body).toMatchObject({
+				id: 1,
+				organizationId: 1,
+				organization: {
+					id: 1,
+					name: 'Example Inc.',
+					employerIdentificationNumber: '11-1111111',
+					createdAt: expectTimestamp,
+				},
+				proposalId: 1,
+				proposal: {
+					id: 1,
+					opportunityId: 1,
+					externalId: '1',
+					versions: [],
+					createdAt: expectTimestamp,
+				},
+				createdAt: expectTimestamp,
+			});
+			expect(after.count).toEqual(1);
+		});
+
+		it('returns 400 bad request when no proposalId is sent', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await insertTestOrganizations();
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					organizationId: 1,
+				})
+				.expect(400);
+			expect(result.body).toMatchObject({
+				name: 'InputValidationError',
+				details: expect.any(Array) as unknown[],
+			});
+		});
+
+		it('returns 400 bad request when no organizationId is sent', async () => {
+			await db.sql('opportunities.insertOne', {
+				title: '🔥',
+			});
+			await insertTestOrganizations();
+			await createProposal({
+				opportunityId: 1,
+				externalId: '1',
+			});
+			const result = await agent
+				.post('/proposals/1/organizations')
+				.type('application/json')
+				.set(authHeader)
+				.send({
+					proposalId: 1,
+				})
+				.expect(400);
+			expect(result.body).toMatchObject({
+				name: 'InputValidationError',
+				details: expect.any(Array) as unknown[],
+			});
+		});
+
+		it('returns 409 Conflict when a non-existent organization is sent', async () => {
```

Comment:
Why a 409 here?

Reply:
It's what we do right now -- there's an issue for changing all of these over to [422 status codes](https://github.com/PhilanthropyDataCommons/service/issues/827), but this PR is already so big that I didn't want to have to refactor that as well.